app/components/Upload/ImageUpload.js

```diff
@@ -46,27 +46,40 @@ type UploadAreaProps = {
   image: ?string,
 };
 
-const FilePreview = ({ file, onRemove, index }: FilePreviewProps) => (
-  <Flex
-    wrap
-    className={styles.previewRow}
-    alignItems="center"
-    justifyContent="space-between"
-  >
-    <div style={{ width: '90%' }}>
-      <TextInput
-        disabled
-        value={file.name}
-        style={{ width: '100%', height: 50 }}
+const FilePreview = ({ file, onRemove, index }: FilePreviewProps) => {
+  const [previewUrl, setPreviewUrl] = useState();
+  useEffect(() => {
+    !previewUrl && setPreviewUrl(URL.createObjectURL(file));
+    return () => {
+      if (previewUrl) URL.revokeObjectURL(previewUrl);
+    };
+  }, [file, previewUrl]);
+
+  return (
+    <Flex
+      wrap
+      className={styles.previewRow}
+      alignItems="center"
+      justifyContent="space-between"
+    >
+      <div style={{ width: '90%', display: 'flex' }}>
+        <img width="60" alt="preview" src={previewUrl} />
+        <div style={{ flex: '1 1 auto' }}>
+          <TextInput
+            disabled
+            value={file.name}
+            style={{ width: '100%', height: 50 }}
+          />
+        </div>
+      </div>
+      <Icon
+        name="close"
```

Comment:
I think a trash icon is more fitting, especially since we're using a close icon at the top with a different functionality

Reply:
This is unchanged in this PR btw.

Thought about this myself. It's also really tiny and hard to click.

Reply:
Done!